src/components/compound/DlSearches/DlSmartSearch/components/DlSmartSearchInput.vue

```diff
@@ -678,6 +679,40 @@ export default defineComponent({
             }
         }
 
+        const onPaste = (e: ClipboardEvent) => {
+            const selection = window.getSelection()
+            if (selection.rangeCount) {
+                let text = (e.clipboardData || (window as any).clipboardData).getData('text')
+                if (text?.length > 0) {
+                    const range = selection.getRangeAt(0)
+
+                    const container = range.startContainer
+                    const offset = range.startOffset
+                    range.setStart(input.value, 0)
+                    const preceedingText = selection.toString()
+                    range.setStart(container, offset)
+
+                    if (
+                        /(IN|[^>]=|,)\s*$/.test(preceedingText) &&
```

Comment:
Can we use the `const operators` ?

Reply:
do you mean like this:
<img width="361" alt="Screenshot 2024-12-20 at 2 34 56 AM" src="https://github.com/user-attachments/assets/1d1acc1a-8a93-4a32-8e3e-af27d77d05c2" />

Reply:
Yes, just that you have these operators in an existing reusable list, don't you ?

Reply:
cant use that list because we only need some of the operators. e g > or <= or EXISTS are irrelevant to strings.

Reply:
Are there other occasions where only these operators are needed ? It might be good to have these operators as one list that we can use where needed, and we'll be able to use it in the whole operators list too.